src/meta/src/hummock/manager/mod.rs

```diff
@@ -654,6 +650,14 @@ where
                 &versioning.current_version,
                 compaction_group_id,
             );
+
+            tracing::trace!(
```

Comment:
Currently we will print each comapct task to info log. For trival move, I think we can also print it to info log?

Reply:
Yes, we can print it to info log, im not sure that how much the trivial_move task accounts for in the whole. I think need a metrics about `trivial_move` in other PR